WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeActions.java

```diff
@@ -161,7 +161,8 @@ public void onErrorResponse(VolleyError volleyError) {
 
         Map<String, String> params = new HashMap<>();
         params.put("keyring_connection_ID", Long.toString(keyringConnectionId));
-        if (!externalUserId.isEmpty()) {
+        // Sending the external id for Twitter connections result in an error
+        if (!externalUserId.isEmpty() && !serviceId.equals(PublicizeConstants.TWITTER_ID)) {
```

Comment:
wdyt about using `TextUils.isEmpty(externalUserId)` and `PublicizeConstants.TWITTER_ID.equals(serviceId)` to avoid potential NPEs?

Reply:
Ha, I have been in Kotlin world for too long, good catch! I'll make the change later today.

Reply:
Addressed in dc812045357ea150161e555b59ff4a30bb0813f8.